rllib/policy/policy_template.py

```diff
@@ -25,7 +25,7 @@
 torch, _ = try_import_torch()
 
 
-# TODO: (sven) Unify this with `build_tf_policy` as well.
+# TODO: Deprecate in favor of directly sub-classing from TorchPolicy.
```

Comment:
when you say subclassing, do you mean that we can build classes that directly inherit from torch policy, as opposed to using the policy builder model?

Reply:
oh wait thats the whole point of the PR